shell/components/Resource/Detail/ResourcePopover/ResourcePopoverCard.vue

```diff
@@ -0,0 +1,144 @@
+<script lang="ts">
+import Card from '@shell/components/Resource/Detail/Card/index.vue';
+import { useStore } from 'vuex';
+import ActionMenu from '@shell/components/ActionMenuShell.vue';
+import { useI18n } from '@shell/composables/useI18n';
+
+export interface Props {
+  resource: any;
+}
+</script>
+
+<script setup lang="ts">
+const emit = defineEmits(['action-invoked']);
+const props = defineProps<Props>();
+const store = useStore();
+const i18n = useI18n(store);
+
+const getGlanceItemValueId = (glanceItem: any): string => `value-${ glanceItem.label }:${ glanceItem.content }`.toLowerCase().replaceAll(' ', '');
+</script>
+
+<template>
+  <Card
+    class="resource-popover-card"
+    :title="'cattle-system'"
+    dropdown-menu-item
+  >
+    <template #heading-action>
+      <ActionMenu
+        :resource="props.resource"
+        :button-aria-label="i18n.t('component.resource.detail.resourcePopoverCard.ariaLabel.actionMenu', { resource: props.resource.nameDisplay })"
+        data-testid="resource-popover-action-menu"
+        @action-invoked="emit('action-invoked')"
+      />
+    </template>
+
+    <div>
+      <div
+        v-for="(glanceItem, i) in props.resource.glance"
+        :key="glanceItem.label"
+        class="row"
+      >
+        <label
+          class="label text-deemphasized"
+          :for="getGlanceItemValueId(glanceItem)"
+        >
+          {{ glanceItem.label }}
+        </label>
+        <div
+          :id="getGlanceItemValueId(glanceItem)"
+          class="value"
+        >
+          <component
+            :is="glanceItem.formatter"
+            v-if="glanceItem.formatter"
+            v-bind="glanceItem.formatterOpts"
+            :id="i === 0 ? 'first-glance-item' : undefined"
+            :value="glanceItem.content"
+            :tabindex="i"
```

Comment:
question: We want to avoid manually assigning tab indexes as a best practice - the structure of our dom nodes should suffice in providing us with the proper tab order. Is there a problem we're trying to solve where `tabindex=0` will not suffice?

Reply:
I had a ton of issues with tabbing and this was just one of the left over items. I removed it and it seems like we're good without it at this point.